syncserver/src/server/mod.rs

```diff
@@ -266,6 +273,14 @@ impl Server {
             &Metrics::from(&metrics),
             blocking_threadpool.clone(),
         )?;
+        let glean_logger = Arc::new(GleanEventsLogger {
```

Comment:
Future optimization would be to have `glean_logger` be an `Option<GleanEventsLogger>` and only instantiate it if `settings.syncstorage.glean_enabled`. That saves us a byte in the `ServerState`.